lib/mongo/server/connection_pool.rb

```diff
@@ -1132,27 +994,27 @@ def check_invariants
         end
       end
 
-      # Close the connections in the given list.
+      # Close the available connections.
       #
       # @param [ Array<Connection> ] connections A list of connections.
       # @param [ Object ] service_id The service id.
-      def close_connections(connections, service_id)
+      def close_available_connections(service_id)
```

Comment:
Changed this to explicitly close available connections because it was only being used on available connections, and I don't want it to be used for other connections in the future, since if it is used on pending or checked_out connections,  the size_cv has to be signaled as well.